libbeat/common/xml.go

```diff
@@ -0,0 +1,25 @@
+package common
+
+import (
+	"github.com/clbanning/mxj/v2"
+)
+
+// UnmarshalXML takes a slice of bytes, and returns a map[string]interface{}.
+// If the slice is not valid XML, it will return an error.
+func UnmarshalXML(body []byte, prepend bool, toLower bool) (obj map[string]interface{}, err error) {
+	var xmlobj mxj.Map
+	// Disables attribute prefixes and forces all lines to lowercase to meet ECS standards
+	mxj.PrependAttrWithHyphen(prepend)
```

Comment:
Yeah this one was currently hardcoded because I couldn't find any reason why we wanted to have all key names starting with a hyphen, it would be easy to change later down the line though.

Reply:
Ok, let's leave it as is by now and we'll see if there are cases that need some other handling :+1:

Reply:
Btw, my thoughts were not about leaving the fields with an hyphen in the final event, but about maybe putting the attributes under a different namespace, so they can be distinguished from children objects.